apps/api/src/auth/auth.service.ts

```diff
@@ -273,4 +282,48 @@ export class AuthService {
       },
     )
   }
+
+  async forgotPass(forgotPasswordDto: ForgotPass) {
```

Comment:
this method should be called sendMailForPasswordChange because this is what it is doing